source/_components/sensor.scrape.markdown

```diff
@@ -36,6 +36,7 @@ Configuration variables:
 - **authentication** (*Optional*): Type of the HTTP authentication. Either `basic` or `digest`.
 - **username** (*Optional*): The username for accessing the website.
 - **password** (*Optional*): The password for accessing the website.
+- **scan_interval** (*Optional*): Defines number of seconds for polling interval (defaults to 60 seconds).
```

Comment:
There is no need to add this. It described [here](https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval) already and is valid for platform which are doing polling.